src/styles/components/_product-media.scss

```
@@ -1,30 +1,86 @@
 .product-media {
-  display: flex;
+  @include flex-center();
   flex-direction: column;
-  align-items: center;
   box-sizing: border-box;
+  height: 450px;
   padding: rem(16px);
+  border-radius: 15%;
 
-  &__header {
-    margin-bottom: rem(24px);
+  @include respond-above('sm') {
+    height: 600px;
   }
 
-  &__title {
-    margin: 0;
+  @include respond-above('md') {
+    height: 430px;
+
+    border-radius: 15%;
+
+    background: $light;
```

How about adding space only when there are more than 1 css properties for each group? 

Do you think this looks cleaner
```suggestion
    height: 430px;
    border-radius: 15%;
    background: $light;
```